packages/generator-single-spa/src/root-config/templates/src/layout/index.ejs

```diff
@@ -1,82 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Layout root configs used to contain the layout definition in the html, but that got moved to a separate file, so now two copies aren't needed.